pyproject.toml

```diff
@@ -1,32 +1,109 @@
 [build-system]
+build-backend = "hatchling.build"
```

Comment:
hatchling is a modern version of setuptools, that uses PEP-621 instead of setup.py/setup.cdg. You shouldn't really care about it or need to use it, modern build frontends (pip or build) make use of PEP-517 to use it, from end user pov like you it's a build implementation detail.

Reply:
But I am still wondering how the version bumping workflow works now. Can you please explain or give me a link about how bumping the version is supposed to work no?